src/libraries/System.Private.CoreLib/src/System/Math.cs

```diff
@@ -1299,6 +1299,18 @@ public static double Round(double value, int digits)
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
         public static double Round(double value, MidpointRounding mode)
```

Comment:
Do we have tests that call this overload with ToEven?  A quick glance at our tests suggest "no", but maybe I'm missing it?

Reply:
Yeah, in `String.Equals` PR I added an explicit test because I made a bug in C# impl by forgetting a nullcheck and all tests still passed 😄 (something like that)

Reply:
On that point, we never inline (even for AggressiveInlining) in debug, right?  Does that mean IsKnownConstant code paths won't ever be covered in debug test runs, only release?

Reply:
> On that point, we never inline (even for AggressiveInlining) in debug, right? Does that mean IsKnownConstant code paths won't ever be covered in debug test runs, only release?

Yes, I agree that all use-cases with `IsKnownConstant` need special tests. I don't expect us to use it a lot thought, I decided to introduce it for `String.Equals` because if I did that opt in JIT itself that would be a lot more changes which are more difficult to maintain.

Reply:
> Is it even possible to test those code paths in debug, though?

I don't see how, I think it's not possible, we can make IsKnownConstant intrinsic "must expand" so it will return true for constants even in Debug mode but it will be useless without general inlining

Reply:
> I don't see how, I think it's not possible

Ok, good to know.